docs/containers.conf.5.md

```diff
@@ -195,6 +195,9 @@ than `0`.
 If you omit the unit, the system uses bytes. If you omit the size entirely,
 the system uses `65536k`.
 
+**tz="local"**
+  Set timezone in container. Takes IANA timezones as well as `local`, which sets the timezone in the container to match the host machine.
```

Comment:
Add 
If not set, then containers will run with the time zone specified in the image.

Reply:
Also could you add an example?

tz="America/New_York"

Reply:
@rhatdan Updated